docs/tutorial.md

```diff
@@ -0,0 +1,285 @@
+
+## What have we just done?
+
+After a successful deployment you should see, either in the dashboard or on the CLI, that you have an HTTP endpoint you can call. Also, if you open the service we just created in your favourite IDE or text editor and look at the contents of the `serverless.yml`, this is what controls pretty much everything in our service. You will notice a section where the functions you have are defined with events attached to them. Also take note that the code that executes when this HTTP endpoint is called is defined in the `handler.js` file in a function called `hello`. If you edit this file then run `serverless deploy` your changes will be pushed to your AWS account and when you next call that endpoint either in the browser or using curl, you should see your changes reflected:
+
+```bash
+curl [your endpoint address]
+```
+
+## Create a new web API endpoint
+
+Now that we have some basics under our belt, let’s expand this further and add some useful endpoints. It would be great to have a POST endpoint so we can add new records to a database. In order to do this we will use an AWS service called DynamoDB that makes having a datastore for Lambda functions quick and easy and very uncomplicated.
+
+## Adding a DynamoDB database
+
+In order to do this, let’s open `serverless.yml` and paste the following at the end of the file:
+
+```yaml
+resources:
+  Resources:
+    CustomerTable:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        AttributeDefinitions:
+          - AttributeName: primary_key
+            AttributeType: S
+        BillingMode: PAY_PER_REQUEST
+        KeySchema:
+          - AttributeName: primary_key
+            KeyType: HASH
+        TableName: ${self:service}-customerTable-${sls:stage}
+```
+
+And lets create a new file in the same folder as the `serverless.yml` called `createCustomer.js` and add the following code to it:
+
+```javascript
+'use strict';
+const AWS = require('aws-sdk');
+
+module.exports.createCustomer = async (event) => {
+  const body = JSON.parse(Buffer.from(event.body, 'base64').toString());
+  const dynamoDb = new AWS.DynamoDB.DocumentClient();
+  const putParams = {
+    TableName: process.env.DYNAMODB_CUSTOMER_TABLE,
+    Item: {
+      primary_key: body.name,
+      email: body.email,
+    },
+  };
+  await dynamoDb.put(putParams).promise();
+
+  return {
+    statusCode: 201,
+  };
+};
+```
+
+You may have noticed we include an npm module to help us talk to AWS, so lets make sure we install this required npm module as a part of our service with the following command:
+
+```bash
+npm install –save aws-sdk
```

Comment:
It should be `--save` instead of `-save`  I believe. Additionally, should we install it as dep instad of dev dep? The SDK will be included, specially that we're using v2 version of the SDK

Reply:
Ah I forgot the second part, let me change that to `--save-dev`